tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -102,8 +102,8 @@ elif [[ $BUILD_DEVICE = "tensorrt"* ]]; then
             IMAGE="$BUILD_OS-cuda11.0-cudnn8.0-tensorrt7.1"
             DOCKER_FILE=Dockerfile.ubuntu_tensorrt7_1
         else
-            # TensorRT container release 20.12
-            IMAGE="$BUILD_OS-cuda11.1-cudnn8.0-tensorrt7.2"
+            # TensorRT container release 21.07
```

Comment:
fyi, this whole path doesn't get exercised anymore. TRT pipelines are all using CentOS now just like the rest of inferencing pipelines.
see: https://github.com/microsoft/onnxruntime/commit/b854f2399dafb379584a0d7aebe42c958ae5aede#diff-18537706fb1a4ba3bd5ec5472648b85149f0ea6abb99eb8192a92607e3a43552

so we may want to remove the entire if TensorRT block  (or if we want to do later, remove these changes)

Reply:
ok let's keep this as you mentioned.
we can set up other pipelines that test ubuntu and pass in -o ubuntu20.04 -d tensorrt 
to exercise the new published tensorrt container image.